### ISSUE TITLE
misspelled extension category name causes trouble

### ISSUE THREAD

**Issue by alexkogan** [ericwinger/Jade#205]
I'm able to add a category in extension format with name that doesn't match loaded package. For 
example I misspelled <*ernei-extensions>.  Attempts to remove or rename given category  fail.
```
a UserDefinedError occurred (error 2318), reason:halt, A package for hybrid package name '*ernei-extensions' was not found.
[] in  ExecBlock0 (RwPrjBrowserTool) >> removeProtocol:fromClassNamed:isMeta: @5 line 16
RwGsImage class >> loadedHybridPackageNamed:ifAbsent: @4 line 14
RwPrjBrowserTool >> removeProtocol:fromClassNamed:isMeta: @10 line 13
RowanErnieTestCase class (Behavior) >> rwRemoveCategory: @8 line 4
[] in  ExecBlock1 (RowanClassService) >> removeCategories: @2 line 7
Array (Collection) >> do: @5 line 10
RowanClassService >> removeCategories: @9 line 6
RowanClassService (RowanService) >> servicePerform:withArguments: @2 line 2
RowanClassService >> servicePerform:withArguments: @4 line 4
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @10 line 12
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1

**Comment by LisaAlmarode** [comment]
Still a problem in 3.0.43

**Comment by ericwinger** [comment]
I'll have code in 3.0.44 that prevents adding a '*' category if the package doesn't exist. Not a perfect solution as it still allows a user to programmatically enter an unremovable category. However, to be more robust, I think we should fix Rowan to be more tolerant of *'d categories without packages.

**Comment by LisaAlmarode** [comment]
This does appear to be working well.  The first time I tried this, I saw errors, perhaps RowanClassService wasn't initialized.  These don't reproduce, but recording just in case. 

ProjectsBrowser:
a ImproperOperation occurred (error 2085), Expected nil to be a Boolean.
ImproperOperation (AbstractException) >> _signalFromPrimitive: @5 line 15
RowanClassService >> addCategory: @4 line 6
RowanClassService (RowanService) >> servicePerform:withArguments: @2 line 2
RowanClassService >> servicePerform:withArguments: @4 line 4
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @10 line 12
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1

Dictionaries Browser: Error dialog, None of an OrderedCollection(a ContainerView a ContainerView) match ''
